src/server/vnparse.c

```diff
@@ -2595,6 +2745,84 @@ add_to_resc_limit_list_as_head(pbs_list_head *phead, resc_limit_t *resc)
 	return (0);
 }
 
+#if !(defined(PBS_MOM) || defined(PBS_PYTHON))
+/**
+ * @brief
+ *	inserts the resource in the resc_limit_t in lexically increasing order of
+ *	resource name
+ *
+ * @param[in]	have - the resc_limit structure.
+ * @param[in]	kv_keyw - resource name.
+ * @param[in]	kv_val - resource value.
+ * @param[in]	execv_f - flag to indicate if the resource was found in execvnode
+ *
+ * @return int
+ * @retval 0	- successful operation.
+ * @retval PBSE_Error	- Error Code.
+ */
+int
+resc_limit_insert_other_res(resc_limit_t *have, char *kv_keyw, char *kv_val, int execv_f)
```

Comment:
Check for NULL-ness of 'have', 'kv_keyw', and 'kv_val'...

Reply:
From all the location of invocations, `have` can never be NULL. For `kv_keyw` and `kv_val` has been handled in the new commit https://github.com/PBSPro/pbspro/pull/1452/commits/8cddc27f70d0dfe1f6738946eee0cfdcda7a311f

Reply:
Can not say "never". I still want that NULL-ness check for 'have'. What if in the future, someone tries to reuse the function and inadvertently pass a NULL for 'have'. Then server will crash.

Reply:
OK, Done, see commit https://github.com/PBSPro/pbspro/pull/1452/commits/6151018baa0b74a7f013bf2e272e4ce090374365